bugfix/1852 - Fix airborne aircraft calling up with "ready to taxi"

cake-pie opened the pull request:
Resolves #1852

The purpose of this pull request is to change `AircraftModel.callUp()` so that the call up message that is used is determined by whether acft is airborne or not, rather than whether it is an arrival or departure acft.

cake-pie committed:
bugfix/1852 - Fix airborne aircraft calling up with "ready to taxi"

n8rzz reviewed (approved):
As usual, if there is room to add a test or two to cover this i'd love to see it. I don't remember the state of tests for the `AircraftModel`, so if it's a pain in the ass don't worry about it.

cake-pie committed:
bugfix/1852 - code gardening

renamed: _updateAircraftControllability is more accurate and disambiguates from AircraftController._updateAircraftVisibility
return earlier if projected, before doing airspace checks

cake-pie commented:
Not covered by any pre-existing tests, and having to set up mocks for cases where the aircraft is inside or outside airspace is more complex than I'd be comfortable tackling.

n8rzz commented:
> Not covered by any pre-existing tests, and having to set up mocks for cases where the aircraft is inside or outside airspace is more complex than I'd be comfortable tackling.

Totally fair, don't worry about it then. Thanks for checking!
